code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -45,6 +45,7 @@
 	metabolization_rate = 0.125 * REAGENTS_METABOLISM
 	ph = 8
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+	addiction_types = list(/datum/addiction/nicotine = 50) // pretty dang addictive
```

Comment:
👀 

time to use incredibly tiny microdoses of nicotine to get people addicted to cigarettes

Reply:
like, for reference here, addiction_gain_threshold is 600, and nicotine has an incredibly low metabolization rate. correct me if I'm wrong here, but I think that even 0.5u of nicotine should be well more than enough to get you addicted to nicotine.

Reply:
Yeah, I've been playing around with the numbers, but I also want people to be able to get addicted from normal smoking in a reasonable amount of time. With the numbers now you need to smoke basically an entire cigarette non-stop to get addicted.

Reply:
Yeah, 50 is way too high, going to tone it down.

Reply:
Right, should be a bit more sensible now. 9u of nicotine or 2 Robust cigarettes back to back will cause addiction at neutral mood.